contracts/TablelandTables.sol

```diff
@@ -144,7 +152,7 @@ contract TablelandTables is
             revert Unauthorized();
 
         uint256 querySize = bytes(statement).length;
-        if (querySize > QUERY_MAX_SIZE)
+        if (querySize >= QUERY_MAX_SIZE)
```

Comment:
@anudit can you explain this one a bit?

Reply:
When the > operator is used, the EVM needs to perform a subtraction operation to determine if the condition is true. For example, if we have the statement if (a > b), the EVM needs to subtract b from a to determine if a is greater than b.

On the other hand, when the >= operator is used, the EVM only needs to perform an equality check followed by a subtraction if the equality check passes. For example, if we have the statement if (a >= b), the EVM checks if a is equal to b and then performs a subtraction only if a is greater than b. Goes the cheaper route first.